sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CustomFormModel.java

```diff
@@ -55,21 +55,21 @@
      *
      * @param modelId Model identifier.
      * @param modelStatus Status of the model.
-     * @param createdOn Date and time (UTC) when the model was created.
-     * @param lastUpdatedOn Date and time (UTC) when the status was last updated.
-     * @param subModels List of sub models.
+     * @param requestedOn Date and time (UTC) when the training model was requested.
+     * @param completedOn Date and time (UTC) when the model training was completed.
+     * @param submodels List of sub models.
```

Comment:
Maybe elaborate a little more for the docstring for submodels? In python we say "A list of submodels that are part of this model, each of which can recognize and extract fields from a different type of form."